include/common/bitdepth.rs

```diff
@@ -422,6 +422,48 @@ where
     }
 }
 
+/// Declare and select a [`BitDepth`]-dependent `extern "C" fn`.
+///
+/// # Args
```

Comment:
Version without the decl_fn is not documented, should it be used?

Reply:
Oh, yes, I forgot about it.

Reply:
Fixed in ce61e08.